documentation/testing/test-plan.md

```diff
@@ -1,106 +1,132 @@
+UAT will be conducted in a dedicated testing environment that closely mirrors the production environment with real data. We'll use actual data files to check if the system works correctly and does the right calculations by writing automated tests/scripts. The schedule of UAT is yet to be defined. All team will be involved in UAT with tech team focusing on writing autoamted scripts and rest team on manual testing. 
 
 ## Test Data
+
 ### Required Test Data
 Synthetic test data will be utilised to simulate various scenarios.
 
 ### Data Generation
 The Data Engineers on the project will collaborate to generate relevant test data, facilitating thorough testing.
 
 ## Risk Management
+
 ### Potential Risks
 A potential risk is the time constraint within sprints for adequately covering automated tests, manual testing, and cross-browser testing.
 
 ### Risk Mitigation
 The entire team will collectively assess and address risks during sprint planning, ensuring a proactive approach to risk mitigation.
 
 ## Defect Management
-Defects will be reported via DevOps tickets, closely linked to the respective user stories.
+Defects will be reported via DevOps task, closely linked to the respective user stories.
 
 ## Testing Team
 The primary tester, Faizan Ahmad, will lead testing efforts. The entire team will share the responsibility for maintaining overall product quality.
 
 ## Communication Plan
-Communication will be facilitated through DfE email or Teams. Technical discussions will primarily take place within DevOps tickets.
-
-## Entry Criteria
-Entry criteria to start the testing of a particular feature will be dependent on the following. The automated tests for the new feature will be written along with the dev work when possible.  
-
-| No | Entry Criteria                     | 
-|----|------------------------------------|
-| 1  | dev ticket has been completed      |
-| 2  | all integrations tests are passing |
-| 3  | all unit tests are passing         |
-| 4  | functional e2e tests are passing   |
-## Exit Criteria
-The testing phase will be considered complete when all acceptance criteria have been successfully implemented and tested, automated tests have been integrated, and accessibility testing has been comprehensively conducted.
-
-### Pass and Fail Criteria
-| Test Status | Definition                                            | 
-|-------------|-------------------------------------------------------|
-| Passed      | Test Outcome meets the Expected Test Outcome          | 
-| Failed      | Test Outcome DOES NOT meets the Expected Test Outcome |
-| Blocked     | Existing Defect Blocks Test Execution                 |      
-| Untested    | Yet to be Tested                                      |       
-
+Communication will be facilitated through Teams or Slack. Discussions will primarily take place within DevOps tickets for the sake of visibility and back tracing.
```

Comment:
This feels contradictory? 

> Communication will be facilitated through Teams or Slack. Actions and decisions will be recorded on the DevOps tickets to ensure both visibility and traceability.

Reply:
Good spot. updated it now so that decisions and outcomes are logged in ticket.